README.md

```diff
@@ -5,6 +5,12 @@ QChat client in QField, let's chat with your GIS tribe on the field !
 - See [QTribu](https://github.com/geotribu/qtribu): QGIS plugin and QChat client in QGIS
 - QChat client compatible with [gischat backend](https://github.com/geotribu/gischat)
 
+## Installation
+
+You can follow [the QField documentation](https://docs.qfield.org/how-to/plugins/#application-plugins) regarding plugin installation.
+
+The URL to install the `QChat` plugin is : `https://github.com/geotribu/qchat-qfield-plugin/releases/latest/download/qfield-plugin-qchat.zip`.
```

Comment:
typo and make it copiable through web ui ;)

````suggestion
The URL to install the `QChat` plugin is:

```
https://github.com/geotribu/qchat-qfield-plugin/releases/latest/download/qfield-plugin-qchat.zip
```
````